resources/js/pages/messages.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+
+import { SEO } from '@/common/components/SEO';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { MessagesRoot } from '@/features/messages/components/+root';
+
+const Messages: AppPage = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SEO title={t('Messages')} description="Your inbox messages" />
+
```

Comment:
Would it be appropriate to include a user breadcrumb?

We'd definitely want some sort of discriminator if we open up the team account message lists in the future.

Reply:
I think a user breadcrumb makes sense as a way to potentially solve this problem. Added in latest:

![Screenshot 2025-01-25 at 2 59 17 PM](https://github.com/user-attachments/assets/7b153251-99bc-4aca-a9cf-02f38c7c08f3)

Reply:
I was kind of expecting "All Users > User > Messages", but this does seem consistent (at least for now) with the old messages that are just "Messages > New Message" or "Messages > Subject". Maybe we can revisit when they get converted.

Reply:
I did waffle around on this a bit with the same thought.

Ultimately, I was afraid All Users as the base nav could be an accessibility issue, as it may imply I can navigate to N users' inboxes, whereas realistically for 99.9% of users this is going to be completely locked down.